2.6/s2i/bin/run

```diff
@@ -0,0 +1 @@
+/bin/run-mongod
```

Comment:
@bparees What do you mean by this?

In majority of s2i based images `run` script is used as default image CMD - it is not a mongodb case (it has different default CMD). So I've added this link mainly for "compatibility" reasons and expected that it won't be used. Is the assumption wrong, is something using specially `/usr/libexec/run` ?

Or is this comment related to https://github.com/sclorg/mongodb-container/issues/246 ?